src/shared/components/Posts/components/PostsAbout/PostAbout.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+//interfaces
+import { IPostResponse } from '../../interfaces/IPostsResponse';
+//hooks
+import usePosts from '../../hooks/usePosts';
+//components
+import { PageLayout } from 'shared/components/PageLayout';
+import { SinglePost } from '../SinglePost';
+import { Loader } from 'shared/components/Loader';
+//styles
+import styles from './PostAbout.module.scss';
+const PostAbout: FC = ({ children }) => {
+    const { id } = useParams();
+    const [post, setPost] = useState<IPostResponse>();
+
+    const { postsData, arePostsLoading } = usePosts();
+
+    useEffect(() => {
+        const getSinglePost = postsData.find((post) => {
```

Comment:
this could be outside of useEffect and just pasted in <SinglePost post={post} />